code/game/objects/items/melee/misc.dm

```diff
@@ -60,8 +60,8 @@
 	//very imprecise
 
 /obj/item/melee/sabre
-	name = "officer's sabre"
-	desc = "An elegant weapon, its monomolecular edge is capable of cutting through flesh and bone with ease."
+	name = "officer's sabre" //SKYRAT EDIT - Buffed in modular_skyrat/modules/modular_weapons/code/melee.dm
```

Comment:
Jungle. Why did you put this on the `name` instead of just the item path.
You're a psychopath. Lunatic. Insane.

Reply:
shrimply built different